BUILDING.md

```diff
@@ -0,0 +1,57 @@
+Building sailjail
+=================
+Sailjail uses [Meson build system](https://mesonbuild.com/). To build it
+use the following commands:
+
+    meson setup build
+    meson compile -C build
+
+See [the next section](#useful-setup-options) for more tips.
+
+Useful setup options
+--------------------
+Some useful options for `meson setup`:
+
+| option                   | explanation                             |
+| -----------------------: | :-------------------------------------  |
+| -Db\_coverage=true       | Build with support for coverage reports |
+| -Dlibdbusaccess=disabled | Build without libdbusaccess support     |
+
+Running unit tests
+------------------
+Unit tests can be run with meson:
+
+    meson test -C build
+
+After that you can also generate a coverage report if you built with
+coverage reports support:
+
+    ninja coverage -C build
+
+It will print you URI to the generated report, assuming that you have
+the commands to build the reports in your system.
+
+Sailjail daemon diagram
+-----------------------
+There is an architecture diagram generated with graphviz and a png
+version of it can be generated with `sailjaild.png` target. You can find
+it in _build/daemon/_ after building it:
+
+    meson compile sailjaild.png -C build
```

Comment:
This one caused me quite a bit of trouble as we don't have graphviz available for `sb2`I couldn't find a solution without editing the build file to make it build outside of `sb2`.

Reply:
I built it outside our SDK. Works great with `-Dlibdbusaccess=disabled`.

Reply:
That's the solution; thank you.